files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -220,6 +235,11 @@ public function stream_flush() {
 			return false;
 		}
 
+		if ( 'r' === $this->mode ) {
+			// No writes in 'read' mode
+			return false;
```

Comment:
Should this `trigger_error()` too?

Reply:
I originally did not think so but a `trigger_error()` could help with debugging so I'll add that in.

Reply:
Changes made in f60d94b3aba76cc7357adb618ed9e13448ab9063
I've also added the same check to `stream_write()`